SwrveSDK/SwrveUtils.brs

```diff
@@ -1,67 +1,173 @@
-' Used in the main thread and in the Render thread. 
-
-'Util function to log strings'
-function SWLog(msg as String)
-	if GetGlobalAA().global.SwrveDebug
-		print "[SwrveSDK] " + msg
-	end if
+' Used in the main thread and in the Render thread.
+function SwrveConstants() as Object
```

Comment:
Moved to the utils to make the implementation of Swrve easier on devs. Just one less file they need to remember to import.